DEVELOPER_GUIDE.md

```diff
@@ -227,6 +228,10 @@ $ yarn opensearch snapshot --P https://repo1.maven.org/maven2/org/opensearch/plu
 
 Note - if you add the [`security` plugin](https://github.com/opensearch-project/security), you'll also need to [configure OpenSearch Dashboards for security](#configure-opensearch-dashboards-for-security).
 
+### Plugin development
+
+[osd-plugin-generator]: (https://github.com/opensearch-project/OpenSearch-Dashboards/tree/main/packages/osd-plugin-generator)
```

Comment:
I dont think this works as you expect. This is not the correct format for markdown links. Markdown links are formatted as `[Link name](URL)`, no colon in between. Also can you add some content in here about the link you are pointing the users to?

Reply:
Done, could you please review my changes?